gravitee-am-management-api/gravitee-am-management-api-service/src/test/java/io/gravitee/am/management/service/DomainNotificationServiceTest.java

```diff
@@ -227,6 +227,21 @@ public void shouldNotifyUserFromGroup() throws Exception {
 
         verify(notifierService, times(11)).register(any(), any(), any());
         verify(userService, never()).findById(any(), any(), any());
+    }
+
+    @Test
+    public void shouldNotNotifyIfGroupIsEmpty() throws Exception {
+        final Membership member = new Membership();
+        member.setMemberType(MemberType.GROUP);
+        member.setMemberId("groupId");
+        when(membershipService.findByCriteria(eq(ReferenceType.DOMAIN), eq(DOMAIN_ID), any())).thenReturn(Flowable.just(member), Flowable.empty());
+        when(groupService.findMembers(any(), any(), any(), anyInt(), anyInt())).thenReturn(Single.just(new Page<>(null, 0, 0)));
+
+        cut.registerCertificateExpiration(certificate);
 
+        Thread.sleep(1000); // wait subscription execution
+
+        verify(notifierService, never()).register(any(), any(), any());
+        verify(userService, never()).findById(any(), any(), any());
```

Comment:
```suggestion
        verify(notifierService, timeout(1000).never()).register(any(), any(), any());
        verify(userService, never()).findById(any(), any(), any());
```

Reply:
For your information `timeout(x).never()` is no longer supported : cf https://github.com/mockito/mockito/commit/dff5510cb3e97dae414e58ba85825ad01b273c72#diff-975688ee2ea34011f98ebb3c72513a6a391f5853d8e87cd9b44f0a9c0d51985c

Reply:
Fair enough